crates/cheatnet/tests/cheatcodes/spoof.rs

```diff
@@ -175,29 +284,33 @@ fn spoof_simple() {
         max_fee_before,
         signature_before,
         _,
+        resource_bounds_before,
+        tip_before,
+        paymaster_data_before,
+        nonce_data_availabilty_mode_before,
+        fee_data_availabilty_mode_before,
+        account_deployment_data_before,
     ) = call_mock_checker_getters(
         &mut blockifier_state,
         &mut cheatnet_state,
         &contract_address,
     );
 
-    let version = None;
-    let account_contract_address = None;
-    let max_fee = None;
-    let signature = None;
-    let transaction_hash = Some(Felt252::from(123));
-    let chain_id = None;
-    let nonce = None;
-
     cheatnet_state.start_spoof(
         CheatTarget::One(contract_address),
-        version,
-        account_contract_address,
-        max_fee,
-        signature,
-        transaction_hash,
-        chain_id,
-        nonce,
+        None,
+        None,
+        None,
+        None,
+        Some(Felt252::from(123)),
```

Comment:
Please add a comment that this line specifies `transaction_hash` 🙏

Reply:
The tests here are a mess anyway, so maybe let's just try make it better in #1331